tests/unit/runtimes/node/utils/node_version.js

```diff
@@ -11,7 +11,7 @@ test('getNodeVersion', (t) => {
   t.is(getNodeVersion('nodejs8.x'), 8)
   t.is(getNodeVersion('12.x'), 12)
   t.is(getNodeVersion('8.x'), 8)
-  t.is(getNodeVersion('node14'), DEFAULT_NODE_VERSION)
+  t.is(getNodeVersion('node16'), DEFAULT_NODE_VERSION)
```

Comment:
Can we add a line here to check that 14.x still works.

Reply:
Ok, I will take advantage of the problems reported by the bots to add this

Reply:
Solved [here](https://github.com/netlify/zip-it-and-ship-it/pull/1138/commits/27835289778d5f3a4aff61b332ec986c4df69b18)